index.bs

```diff
@@ -542,6 +542,19 @@ When the <dfn method for="XRFrame">getHitTestResultsForTransientInput(|hitTestSo
 Geometric primitives {#geometric-primitives}
 ====================
 
+XRRayDirectionInit {#xr-ray-direction-init-dictionary}
+--------------------
+
+An {{XRRayDirectionInit}} dictionary represents a direction vector to be passed to the {{XRRay(origin, direction)}} constructor.
+
+<script type="idl">
+dictionary XRRayDirectionInit {
+  double x = 0;
+  double y = 0;
+  double z = -1;
```

Comment:
I could add a `w` coordinate and explicitly ensure it's `0` in the constructor in case someone passes down a DOMPoint with a nonzero `w`, but I don't see this as being particularly important and might introduce additional friction.

Reply:
I think it would be good to catch this particular case and throw - if the application uses homogeneous coordinates and passes something with non-zero `w` as a direction, that's a bug that we should not auto-fix here.

Reply:
Done.